llarp/util/fs.hpp

```diff
@@ -8,12 +8,15 @@
 #define PATH_SEP "/"
 #endif
 
-#ifndef LOKINET_USE_CPPBACKPORT
-#include <experimental/filesystem>
-namespace fs = std::experimental::filesystem;
-#else
+#ifdef WIN32
```

Comment:
why are we using the Microsoft-specific `<filesystem>`

Reply:
that’s not specific to MS. It’s just on windows we use C++17 which has native `<filesystem>` so that’s what we use

Reply:
oh
see MSVC 2015+ has two `<filesystem>` headers, and i actually kept throwing exceptions with the pre-std version, so `<experimental/filesystem>` is still used in MSVC